.env

```diff
@@ -0,0 +1,2 @@
+GMAIL_USERNAME = blogpassreset@gmail.com
```

Comment:
This is sensitive data, it shouldn't be in the repo. You could create an `env.sample` file to give an example of what the `.env` should look like (also include instructions in the README)

Reply:
I understand, I will make the change to not compromise sensitive data, I will add the .env file to the .gitignore file.

Reply:
remove the `.env` file from the repo